v22.1/use-cloud-storage-for-bulk-operations.md

```diff
@@ -225,6 +225,85 @@ A custom root CA can be appended to the system's default CAs by setting the `clo
 
 </section>
 
+## Storage permissions
+
+This section describes the minimum permissions required to run CockroachDB bulk operations. While we provide the required permissions for Amazon S3 and Google Cloud Storage, the provider's documentation provides detail on the setup process and different options regarding access management.
+
+Depending on the actions a bulk operation performs, it will require different access permissions to a cloud storage bucket. 
+
+This table outlines the actions that each operation performs against the storage bucket:
+
+Operation                   | Permission                   | Description                                                                     
+----------------------------+------------------------------+---------------------------------------------------
+[Backup](backup.html)       | Write<br>List                  | Write: Backups write the backup data to the bucket/container. During a backup job, a `BACKUP CHECKPOINT` file will be written that tracks the progress of the backup.<br> Read: Backups need list access to the files already in the bucket. If the backup is [paused](pause-job.html), or there is a retryable error, the backup will read the `BACKUP CHECKPOINT` file.
```

Comment:
It is correct that Delete is no longer needed, but I wonder if we want to call out somewhere that if the user doesn't provide delete, then we can't cleanup checkpoints on failure. So it is a small tradeoff.

The description here describes why we need to be able to read, but Get isn't included in the permission list or the example policies below.

Reply:
Yes I can add a note on that tradeoff, I think that's a great idea. If you had delete permission, would the checkpoint files only be deleted on failure? There wouldn't be clean up of the checkpoint files if the backup is successful... and that's because we only write multiple if there is a pause/retryable error and we don't overwrite?

On the second point, that was my bad, I had "Read" here but that should have been "List" in the description column. On testing this, `Get` wasn't required for backups in v22.1.

Reply:
Update, I added a draft note to v22.1. Looking at it though, I think I need to define that this is on failure, if you can confirm?

Reply:
> On testing this, Get wasn't required for backups in v22.1.

I think Get is required in v22.1.  For example, if you pause and resume a backup, you need Get in order to read the backup checkpoint.